docs/admin/upgrading_to_1.7.0.md

```diff
@@ -0,0 +1,23 @@
+# Upgrading to 1.7.0
+
+This note describes the necessary steps to upgrade to Stellio 1.7.0
+
+## Contact entity service
+
+Stellio now allows to send periodic notifications, for this it's necessary to add new variables to contact the entity service in the docker-compose. 
```

Comment:
you can also refer to the spec here: e.g., `timeInterval` property for subscriptions as defined in section ...

Reply:
I do this  -> You can refer to the spec : `timeInterval` property for subscription as defined in section `5.2.11`

Reply:
yeah, something like that